versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.12.20, < 0.14"
+  required_version = ">= 0.12.20, < 0.16"
```

Comment:
missing ocmmit and test against 0.14.

Reply:
support, test and commit added in 8fe2990